web/src/people.js

```diff
@@ -1282,6 +1282,30 @@ export function report_late_add(user_id, email) {
     }
 }
 
+function make_user(partial_user) {
+    return {
+        user_id: partial_user.user_id ?? -1,
```

Comment:
The `user_id`, `email`, and `full_name` parameters need to be required parameters.